runtime-decompiler/src/main/java/org/jrd/frontend/PluginMangerFrame/PluginConfigurationEditorController.java

```diff
@@ -263,8 +263,9 @@ private void removeWrapper(DecompilerWrapperInformation wrapperInformation) {
         pluginManager.deleteWrapper(wrapperInformation);
         configPanelHashMap.remove(wrapperInformation);
         updateWrapperList(pluginManager.getWrappers());
+
         if (wrapperJList.getModel().getSize() == 0) {
-            view.dispose();
+            view.clearConfigPanel();
             return;
         }
```

Comment:
Instead of freeing, when there are no plugis, you are now insisting that  there is exactly one,  and removing all components. Can you elaborate a bti more on this change? Why so?

Reply:
> insisting that there is ... one
The assert is there to guide future contributors to the project of when this method is supposed to be used - when the last plugin is being deleted.

> there is exactly one
Exactly one plugin-bound JPanel, containing all those text fields. Note that at this point, the plugin's config file has been deleted, the plugin's JList listing has been removed, and all that remains to be done is to delete the GUI elements associated with the plugin - which is what this method does, instead of the previous, rather crude way of just closing the Plugin config window.